re-frisk/src/re_frisk/ui/events.cljs

```diff
@@ -1,8 +1,9 @@
 (ns re-frisk.ui.events
-  (:require-macros [re-frisk.inlined-deps.reagent.v1v0v0.reagent.ratom :refer [reaction]])
+  (:require-macros [re-frisk.inlined-deps.reagent.v1v2v0.reagent.ratom :refer [reaction]])
   (:require
-   [re-frisk.inlined-deps.reagent.v1v0v0.reagent.core :as reagent]
-   [re-frisk.inlined-deps.reagent.v1v0v0.reagent.dom :as rdom]
+   [react]
```

Comment:
could you pls elaborate on why [react] is needed?

Reply:
Yes, basically I'm replacing the call to `rdom/dom-node`, because it ends up callling `findDOMNode` which is deprecated. There's more info [here](https://18.react.dev/reference/react-dom/findDOMNode). Although I didn't test it.

In fact, I see there's several shadow builds. How can I test the project? I tried running `shadow-cljs watch dev-remote client` and `shadow-cljs run re-frisk-remote.core/start`. But when I press buttons in 3001 I don't see any change in 3002.